addons/survey/views/survey_question_views.xml

```diff
@@ -162,7 +162,8 @@
                             </group>
                         </page>
                         <page string="Description" name="survey_description">
-                            <field name="description" widget="html"/>
+                            <field name="description" widget="html"
+                                   placeholder="e.g. Guidelines, instructions, picture, ... to help attendees answer"/>
```

Comment:
No `&quot;` for this one?

Reply:
Just read the chatter history, should have started there, ignore this comment ^^